File: Anul II/Medii de proiectare si programare/Laborator/lab07/Ubb.BikeContest.Networking/Response/GetParticipantsByTeamResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ubb.BikeContest.Model;

namespace Ubb.BikeContest.Networking.Response
{
    [Serializable]
    public class GetParticipantsByTeamResponse : IResponse
    {
        private readonly List<Participant> teamParticipants;

        public GetParticipantsByTeamResponse(List<Participant> teamParticipants)
        {
            this.teamParticipants = teamParticipants;
        }

        public List<Participant> Participants { get { return teamParticipants; } }
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab08/Java/networking/src/main/java/com/ubb/protocol/response/GetParticipantsByTeamResponse.java
package com.ubb.protocol.response;

import com.ubb.model.Participant;

import java.util.Collection;

public class GetParticipantsByTeamResponse implements Response {
    private final Collection<Participant> participants;

    public GetParticipantsByTeamResponse(Collection<Participant> participants) {
        this.participants = participants;
    }

    public Collection<Participant> getParticipants() {
        return participants;
    }
}
